docs/hello_nf-core/00_orientation.md

```diff
@@ -33,6 +33,29 @@ cd hello-nf-core/
 
 Now let's have a look at the contents of this directory.
 
+## What you'll learn
+
+This training course teaches you the core concepts for building nf-core-style pipelines.
+We can't cover everything—nf-core pipelines have many features and conventions developed by the community over years—but we focus on the essential concepts that will help you get started and understand how nf-core works.
+
+First, you'll **run an existing nf-core pipeline** to understand how they're structured and what makes them different from basic Nextflow workflows.
```

Comment:
Running the pipeline doesn't really help the user understand how they're structured, it literally just helps them understand how to run a pipeline. I feel like the AI is making things up again here.

Reply:
Tweaked!